docs/multicluster/user-guide.md

```diff
@@ -630,6 +633,98 @@ Normally, `podCIDRs` should be the value of `kube-controller-manager`'s
 will not be enabled. If you use `kubectl edit` to edit the ConfigMap, then you
 need to restart the `antrea-mc-controller` Pod to load the latest configuration.
 
+## Multi-cluster Stretched NetworkPolicy
+
+Network Policy can be enforeced on cross-cluster traffic in a ClusterSet in the following ways:
```

Comment:
Should we highlight somewhere about that Stretched NetworkPolicy only works on Geneve mode?